src/test/java/uk/gov/ons/ctp/integration/envoycuc/mockclient/MockLimiter.java

```diff
@@ -0,0 +1,250 @@
+package uk.gov.ons.ctp.integration.envoycuc.mockclient;
```

Comment:
This class now does all the work and is identical to the one in my branch of the mock-rate-limiter. 
So I'm going to put this class in a common place, and Phil wants me to merge this class into master of that project so whan I do, that I'll write some unit tests for it